common/config/rush/common-versions.json

```diff
@@ -69,7 +69,7 @@
     // @azure/event-processor-host is on a much lower major version
     "@azure/ms-rest-nodeauth": ["^0.9.2"],
     // Idenity is moving from v1 to v2. Moving all packages to v2 is going to take a bit of time, in the mean time we could use v2 on the perf-identity tests.
-    "@azure/identity": ["^2.0.0-beta.1", "2.0.0-beta.3", "~1.3.0"],
+    "@azure/identity": ["2.0.0-beta.2", "2.0.0-beta.3", "~1.3.0"],
```

Comment:
Should we use beta 3 now that we confirmed that beta 2 will break in browsers?

Reply:
I'll let you folks decided that. If needed please send a separate PR

Reply:
Both beta 2 and beta 3 are listed here, so you should be fine regardless of what you choose to use

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/15057